person.rb

```diff
@@ -7,6 +7,7 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ] Getters and Setters (attr_accessor) for @rentals should be present, please fix that.